packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
+## v2.0.0 (May 21, 2024)
+- Add Shopper Stores hooks [#1788](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1788)
```

Comment:
let's move this to the section for `v2.0.0-dev`, we use this for the to be released items.